src/scss/site/_variables.scss

```diff
@@ -263,6 +263,7 @@ $spacing-160: $layout-2xl;
 // text variables
 $p-default-max-width: 75ch;
 $p-lead-default-max-width: 55ch;
+$p-max-width: 100%;
```

Comment:
If we do decide to add this selector, I feel it should this be named `$p-full-width` :-)